src/api.js

```diff
@@ -659,13 +668,6 @@
       });
   };
 
-  Contact.prototype.destroy = function (callbacks) {
```

Comment:
We shouldn't remove this. Deprecate this as in the documentation.  
noop and log error (or warn) on use as @ctwomblyamzn suggested on #334